ui/raidboss/data/07-dt/raid/r2s.ts

```diff
@@ -335,6 +499,82 @@ const triggerSet: TriggerSet<Data> = {
       netRegex: { id: '91AA', source: 'Honey B. Lovely', capture: false },
       response: Responses.bigAoe(),
     },
+    {
+      id: 'R2S Beeloved Venom Tracker',
+      type: 'GainsEffect',
+      // F5C: Alpha, F5D: Beta
+      // durations are 12s, 28s, 44s, and 62s
+      netRegex: { effectId: ['F5C', 'F5D'] },
+      run: (data, matches) => {
+        const type = matches.effectId === 'F5C' ? 'alpha' : 'beta';
+        if (data.me === matches.target)
+          data.beelovedType = type;
+
+        const duration = parseFloat(matches.duration);
+        const orderIdx = beelovedDebuffDurationOrder.indexOf(duration);
+        if (orderIdx === -1) // should not happen
+          return;
+        data.beelovedDebuffs[type][orderIdx] = matches.target;
+      },
+    },
+    {
+      id: 'R2S Beeloved Venom Player Merge',
+      type: 'GainsEffect',
+      netRegex: { effectId: ['F5C', 'F5D'] },
+      condition: Conditions.targetIsYou(),
+      delaySeconds: (_data, matches) => parseFloat(matches.duration) - 9,
```

Comment:
Pure subjective suggestion:

Here (and below for other), 9s might be pushing the timing a bit too tight?

The exact timings are:
12s - initial debuffs
16.5s - Time between initial debuffs and first raidwide
7s - Duration of magic vuln from merging with other player

This only leaves a 6.5 second window for players to react to the callout and move to merge with the other player, and have the server actually count them as merging.

It might make sense to have this be 10s instead of 9s? Possibly only for the first merge?

Reply:
> Pure subjective suggestion:
> 
> Here (and below for other), 9s might be pushing the timing a bit too tight?
> 
> The exact timings are: 12s - initial debuffs 16.5s - Time between initial debuffs and first raidwide 7s - Duration of magic vuln from merging with other player
> 
> This only leaves a 6.5 second window for players to react to the callout and move to merge with the other player, and have the server actually count them as merging.
> 
> It might make sense to have this be 10s instead of 9s? Possibly only for the first merge?

Yeah, I definitely think this could be adjusted.  I seem to remember some concern about merging too early (maybe just allowing time to heal up from the raidwide? Can't recall if the raidwide also inflicts a vuln) -- you probably have a better perspective on that as a healer than I do though.

I wasn't able to test this in-game tonight -- hopefully tomorrow at some point -- so let me play with the timing a little and see what feels comfy.

(As an aside, this prompted me to finally get around to #358, so if that lands, I could see it being applied here or added in a later PR to help give a better timeframe to players.)

Reply:
The raidwide doesn't inflict a vuln, so it's safe to merge as soon as party health/shields/mits allow it after the raidwide.

Reply:
I haven't been using triggers on floors 1/2 for various reasons, but as a shot caller, I agree that a 10s warning would be better here. People are sometimes *really* slow to move in when it's their turn, so "merge soon" a little earlier seems better to me, as opposed to "merge now".